components/infobox/wikis/rocketleague/infobox_league_custom.lua

```diff
@@ -188,19 +188,15 @@ function CustomLeague.parseShowHeadToHead(args)
 end
 
 function CustomLeague:addToLpdb(lpdbData, args)
-	lpdbData['game'] = 'rocket league'
-	lpdbData['patch'] = args.patch
-	lpdbData['participantsnumber'] = args.team_number or args.player_number
+	lpdbData.game = 'rocket league'
```

Comment:
not a fan of this line at all, but oos

Reply:
I thought i could fix these while at it. I was surprised by the hardcoded storage as well, if that is what you mean

Reply:
yeah

Reply:
There are multiple questionable things in that infobox
https://github.com/Liquipedia/Lua-Modules/blob/main/components/infobox/wikis/rocketleague/infobox_league_custom.lua#L175

Reply:
lol